cmd/admin-user-svcacct.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * MinIO Client (C) 2021 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import "github.com/minio/cli"
+
+var adminUserSvcAcctSubcommands = []cli.Command{
+	adminUserSvcAcctAddCmd,
+	adminUserSvcAcctListCmd,
+	adminUserSvcAcctRemoveCmd,
+	adminUserSvcAcctInfoCmd,
+	adminUserSvcAcctSetCmd,
+	adminUserSvcAcctEnableCmd,
+	adminUserSvcAcctDisableCmd,
+}
+
+var adminUserSvcAcctCmd = cli.Command{
+	Name:            "svcacct",
```

Comment:
I think this one should be `serviceaccount` with an alias to `sa` for simplicity